src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -896,4 +896,7 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>Package content hash validation failed for {0}. Expected: {1} Actual: {2}</value>
     <comment>0 - package ID, 1 - expected Sha, 2 - actual sha</comment>
   </data>
+  <data name="Error_CentralPackageVersions_VersionsNotAllowed" xml:space="preserve">
+    <value>Projects that use central package version management should not define the version on the PackageReference items but on the PackageVersion items. The following projects violate this rule.  {0}:{1}.</value>
```

Comment:
@anangaur  please review the message.

Reply:
This message is always logged for 1 project. 

> The following projects violate this rule.

is probably too verbose. Calling out the violating pacakge references should be enough.

Reply:
Simplified